meteor/sofie-server/business-logic/services/timeline-integration/rundown-timeline-service.ts

```diff
@@ -127,4 +128,17 @@ export class RundownTimelineService implements RundownService {
 		)
 		this.rundownEventEmitter.emitRundownEvent(adLibPieceInsertedEvent)
 	}
+
+	public async deleteRundown(rundownId: string): Promise<void> {
```

Comment:
I have a question: Is it the right place for this method to exist? We're in RundownTimelineService, in a folder called timeline-integration. I understand _timeline_ as a bunch of timeline objects that needs to be gathered from the contents of the rundown and sent out for playout. Deleting a rundown does not touch the timeline in that sense, and should be generally neutral to playout (perhaps with an exception for expectedPlayoutItems, but that's a different story). Even if we did allow removing active rundowns (making them unsynced), it would not affect the timeline.

Reply:
Hmm. In the case that we say it isn't suppose to live on this service, which other service should then house this logic?

It would very likely be a new service, still housed in the business logic layer, as it is us as a business that are imposing a rule on the deletion, that an active one is not allowed to be deleted.

Reply:
It's a good point Krzrysztof. 
It opens up for the question: Should this file be called `rundown-timeline-service` at all? 
The more I have been working on building the timeline, the more I think that the folder "timeline-integration" doesn't makes sense.

I think we should reevaluate the name of this service.